app.rb

```diff
@@ -0,0 +1,42 @@
+require 'sinatra/base'
+require './lib/peep.rb'
+require './lib/user.rb'
+
+class Chitter < Sinatra::Base
+
+  get '/' do
+    erb(:index)
+  end
+
+  get '/peeps/new' do
+    erb(:peeps_new)
+  end
+
+  post '/peeps' do
+    Peep.add(params[:peep_content], Time.now.strftime("%I:%M %p"), User.current_user_id)
+    redirect('/peeps')
+  end
+
+  get '/peeps' do
+    @peep = Peep.all.reverse
+    erb(:peeps)
+  end
+
```

Comment:
Your peep routes are RESTful, but your sign-in/sign-up routes not as much. Remember, REST is a standard, but it's not everything either, and there are many ways to do routes. If you want to look more into this, this is a good place; https://stackoverflow.com/questions/7140074/restfully-design-login-or-register-resources